packages/sanity/src/ui/tooltip/Tooltip.tsx

```diff
@@ -21,15 +12,6 @@ export interface TooltipProps
  *
  * @internal
  */
-export const Tooltip = ({text, ...rest}: TooltipProps) => {
-  return (
-    <UITooltip
-      content={
-        <Text size={1} weight="medium">
-          {text}
-        </Text>
-      }
-      {...rest}
-    />
-  )
+export const Tooltip = ({...rest}: UITooltipProps) => {
```

Comment:
Great job @ninaandal thanks!
I think you will need to update here the props to `ComponentProps<typeof UITooltip>` so you can support the cases that are failing due to ts checks. 

What that does is expose all the props the original component receives, so you can add things like refs